_themes/rtd-blog/social.html

```diff
@@ -1,19 +1,53 @@
-    <div id="mc_embed_signup_scroll">
-    <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required>
-    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_a6a22369cc2b356379cf789ca_a85a83a5a5" tabindex="-1" value=""></div>
-    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+<!-- Begin MailerLite Signup Form -->
+  <div id="mlb2-4710274" class="ml-form-embedContainer ml-subscribe-form ml-subscribe-form-4710274">
+    <div class="ml-form-align-center">
+      <div class="ml-form-embedWrapper embedForm">
+        <div class="ml-form-embedBody ml-form-embedBodyDefault row-form">
+          <div class="ml-form-embedContent" style="">
+            <h3 style="margin-left: 0">Subscribe to our mailing list</h3>
+            <p>Sign up for our blog and we'll send you information about Sphinx and Read the Docs on a regular basis.</p>
+          </div>
+          <form class="ml-block-form" action="https://static.mailerlite.com/webforms/submit/p8b7z2" data-code="p8b7z2" method="post" target="_blank">
+            <div class="ml-form-formContent">
+              <div class="ml-form-fieldRow ml-last-item">
+                <div class="ml-field-group ml-field-email ml-validate-email ml-validate-required">
+                  <input aria-label="email" aria-required="true" type="email" class="form-control" data-inputmask="" name="fields[email]" placeholder="email address" autocomplete="email" style="width: 90%">
+                </div>
+              </div>
+            </div>
+            <div class="ml-form-checkboxRow ml-validate-required">
+              <label class="checkbox"> <input type="checkbox"> <div class="label-description"> <p></p> <p>I wish to subscribe to the Read the Docs newsletter to receive news and updates. I&nbsp;know I&nbsp;can unsubscribe at any time.</p> <p></p> </div> </label>
+            </div>
+            <div class="ml-form-recaptcha ml-validate-required" style="float:left">
+              <style type="text/css">
+                .ml-form-recaptcha{margin-bottom:20px}.ml-form-recaptcha.ml-error iframe{border:solid 1px red}@media screen and (max-width:480px){.ml-form-recaptcha{width:220px!important}.g-recaptcha{transform:scale(.78);-webkit-transform:scale(.78);transform-origin:0 0;-webkit-transform-origin:0 0}}
+              </style>
+              <script src="https://www.google.com/recaptcha/api.js"></script>
+              <div class="g-recaptcha" data-sitekey="6Lf1KHQUAAAAAFNKEX1hdSWCS3mRMv4FlFaNslaD"></div>
+            </div>
+            <input type="hidden" name="ml-submit" value="1">
+            <div class="ml-form-embedSubmit">
```

Comment:
Can we move the Captcha to after submitting the form? Feels really weird & distracting to have in the sidebar for all users.

Reply:
Unfortunately, MailerLite doesn't support that, I've been told. We didn't have a reCAPTCHA previously, so I suppose we can continue without it. Mildly annoying though.

Reply:
Seems we could just have a form that goes to the next page with more details (eg. our own little landing page). I say let's keep it off for now, and we can iterate. I'd like to get this shipped today.